src/main/java/ti4/helpers/StatusHelper.java

```diff
@@ -53,6 +53,11 @@ public static void BeginScoring(GenericInteractionCreateEvent event, Game game,
             messageText += "\n\n" + Helper.getNewStatusScoringRepresentation(game);
         }
 
+        if (game.isOmegaPhaseMode()) {
+            // Show the effects of the Agendas while scoring
+            ButtonHelper.updateMap(game, event, "After Agendas, Round " + game.getRound() + ".");
+        }
+
```

Comment:
In Omega Phase, scoring happens AFTER agendas. Since a lot can change through status cleanup, ACs, agendas, etc, it's nice to see the map before any scoring.